content/departments/product-engineering/engineering/cloud/devops/managed/restore_process.md

```diff
@@ -0,0 +1,41 @@
+# Restore a managed instance when it is completely gone
+
+<span class="badge badge-note">SOC2/CI-110</span>
+
+**For MI 1.1, see the new process [here](v1.1/mi1-1_restore_process.md).**
+
+## Assess what is deleted
+
+Navigate to `sourcegraph-managed-$CUSTOMER` project, and look at existing compute instance. Does the VM still exist?
+
+- No, the VM is gone.
+
+  - Does the data disk `default-$CURRENT_DEPLOYMENT-data-disk` still exists?
+    - Yes, [re-create the VM with existing data disk](##re-create-the-vm-with-existing-data-disk)
+    - No, [re-create the VM with a new data disk from disk snapshot](##re-create-the-vm-with-new-data-disk-from-disk-snapshot)
+
+- Yes, follow [operation guides](./operations.md) to troubleshoot services condition. If unable to recover running services on the VM, fallback to the recovery plan above.
```

Comment:
> fallback to the recovery plan above.

Maybe specify which step specifically - I presume it would be 1. recreate the VM, then 2. recreate the VM with snapshot.